code/game/objects/items/storage/belt.dm

```diff
@@ -90,7 +90,7 @@
 /obj/item/storage/belt/utility/chief/full/PopulateContents()
 	SSwardrobe.provide_type(/obj/item/screwdriver/power, src)
 	SSwardrobe.provide_type(/obj/item/crowbar/power, src)
-	SSwardrobe.provide_type(/obj/item/weldingtool/experimental, src)//This can be changed if this is too much //It's been 5 years
+	SSwardrobe.provide_type(/obj/item/weldingtool/experimental, src)//This can be changed if this is too much //It's been 5 years //Possibly longer
```

Comment:
```suggestion
	SSwardrobe.provide_type(/obj/item/weldingtool/experimental, src)
```

it's time

Reply:
i know you wanted to retain sovl in this PR but some things are no longer meant to last. if you want to leave it as is just remove your `//Possibly longer` append and we'll just move on

Reply:
I understand.

Rest in peace.